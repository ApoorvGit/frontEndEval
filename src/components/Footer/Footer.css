@import url('https://fonts.googleapis.com/css2?family=Big+Shoulders+Inline+Display:wght@700&family=Raleway:wght@100&family=Sofia+Sans:wght@600&display=swap');

footer {
  height: 80px;
  background-color: black;
  width: 100%;
  color: white;
  position: fixed;
  left: 0;
  bottom: 0;
}
.theme {
  display: flex;
  flex-direction: row;
  padding-left: 10%;
  padding-top: 20px;
  align-items: center;
}
.theme > h1 {
  color: white;
  font-family: 'Sofia Sans', sans-serif;
  font-size: 45px;
  font-weight: 400;
}
.color-box {
  width: 35px;
  height: 35px;
  border-radius: 5px;
  margin: 4px;
}
.purple {
  background-color: purple;
}
.blue {
  background-color: blue;
}
.grey {
  background-color: grey;
}
