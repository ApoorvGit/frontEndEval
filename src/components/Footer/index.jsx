import React from 'react';
import './Footer.css';

export default function Header() {
  return (
    <footer>
      <div className="theme">
        <h1>Theme</h1>
        <div className="color-box purple" />
        <div className="color-box blue" />
        <div className="color-box grey" />
      </div>
    </footer>
  );
}
