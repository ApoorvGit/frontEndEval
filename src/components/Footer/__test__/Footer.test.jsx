import { render } from '@testing-library/react';
import React from 'react';
import Footer from '../index';
import '@testing-library/jest-dom';

describe('Footer', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
