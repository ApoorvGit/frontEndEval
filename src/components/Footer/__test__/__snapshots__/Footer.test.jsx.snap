// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<DocumentFragment>
  <footer>
    <div
      class="theme"
    >
      <h1>
        Theme
      </h1>
      <div
        class="color-box purple"
      />
      <div
        class="color-box blue"
      />
      <div
        class="color-box grey"
      />
    </div>
  </footer>
</DocumentFragment>
`;
